website/docs/reference/about-resources.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "About resource types"
+id: "about-resources"
+description: "Learn about the different resource types in dbt, their supported properties and configurations, and how to use them." 
+sidebar_label: "About resource types"
+---
+
+dbt supports different types of resource types. Each one has a set of supported [properties and configurations](/reference/configs-and-properties) within a dbt project, which are key to how they function and integrate within your data project. 
+
+## Resource types table
+
+The following tables describes each resource type, its identifier, and a brief description of its purpose. 
+
+WHERE DO WE LINK TO FOR EACH RESOURCE TYPE? CONTENT UNDER DOCS? OR UNDER RESOURCE CONFIGS?
+
+| Resource type     | Identifier         | Description |
+|-------------------|--------------------|-------------|
```

Comment:
WHERE DO WE LINK TO FOR EACH RESOURCE TYPE? CONTENT UNDER DOCS? OR UNDER RESOURCE CONFIGS?